Tasks/Vadzim/Task7/query12.sql

```diff
@@ -0,0 +1,11 @@
+﻿--Query 12
+UPDATE tempdb.dbo.Product
```

Comment:
it would only update price but it's required to insert new records and delete ones that not exist
try to use MERGE here